Assets/Countly/Plugins/CountlySDK/Services/SessionCountlyService.cs

```diff
@@ -41,7 +41,7 @@ internal SessionCountlyService(CountlyConfiguration configuration, CountlyLogHel
             _monoBehaviour = monoBehaviour;
 
             if (_configuration.IsAutomaticSessionTrackingDisabled) {
-                Log.Verbose("[Countly][CountlyConfiguration] Automatic session tracking disabled!");
+                Log.Verbose("[SessionCountlyService] Automatic session tracking disabled!");
```

Comment:
mb we could add constructor for those

Reply:
@turtledreams what do you think?

Reply:
too much work, could be sth we do if we really have nothing to do

Reply:
lets not give so much detail